docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -144,11 +144,11 @@ From the root of your project, run the following command to compile your contrac
 ```shell
 dune --cmake-build .
 ```
-You will see your contract being compiled. If there are any errors you will see them in the output.
+The contract copmiles. Any errors display in the output. 
```

Comment:
Same here, preference is "you" and "your". 

There's also a type on "copmiles"

Reply:
Oops - fixed spelling

"being compiled" uses a helping verb and an 'ing word. This structure is passive voice
"you will see" also uses a helping verb and is also passive voice

I changed these statements to active voice.

Reply:
Done